packages/block-library/src/heading/edit.native.js

```diff
@@ -98,13 +98,23 @@ class HeadingEdit extends Component {
 
 		const tagName = 'h' + level;
 
-		const accessibilityLabelLevel = __( 'level' ) + ' ' + level + __( '.' );
-		const accessibilityLabelContent = isEmpty( content ) ? __( 'Empty' ) : this.plainTextContent( content );
-
 		return (
 			<View
 				accessible={ ! this.props.isSelected }
-				accessibilityLabel={ __( 'Heading block' ) + __( '.' ) + ' ' + accessibilityLabelLevel + ' ' + accessibilityLabelContent }
+				accessibilityLabel={
+					isEmpty( content ) ?
+						/* translators: accessibility text. %s: heading level. */
```

Comment:
I believe translators comment need to be on the line just before the `__()` call.

Reply:
Whoops, indeed. I should know better, guess it was already late. 🙂

Reply:
I can update this one

Reply:
Awesome, thanks for chiming in!